server/hotrod/src/main/java/org/infinispan/server/hotrod/logging/Log.java

```diff
@@ -27,7 +26,7 @@
  */
 @MessageLogger(projectCode = "ISPN")
 @ValidIdRange(min = 6001, max = 7000)
-public interface Log extends BasicLogger {
+public interface Log extends org.infinispan.server.core.logging.Log {
```

Comment:
Please don't change this. The logging processor unfolds all inherited classes, resulting in huge implementations.

Reply:
Honestly, it seems like a bug to me that we have a server core logger and each of the specific protocol Log implementations. This means you have to have both loggers in the class to handle the case of a shared log message versus a specific protocol one, but I will add the Log instance to each of the classes that need it as I assume we don't want to duplicate the message in each protocol Log instance.